File: PHP/Human.php
<?php
class Human
{
    private $nik = '';
    private $name = '';
    private $gender = '';

    public function __construct($nik = '', $name = '', $gender = '')
    {
        $this->nik = $nik;
        $this->name = $name;
        $this->gender = $gender;
    }

    // Get name
    public function getNik()
    {
        return $this->nik;
    }

    // Set name
    public function setNik($nik)
    {
        $this->nik = $nik;
    }

    // Get name
    public function getName()
    {
        return $this->name;
    }

    // Set name
    public function setName($name)
    {
        $this->name = $name;
    }

    // Get gender
    public function getGender()
    {
        return $this->gender;
    }

    // Set gender
    public function setGender($gender)
    {
        $this->gender = $gender;
    }
}

File: Python/Human.py
class Human:
    def __init__(self, nik="", name="", gender=""):
        self.nik = nik
        self.name = name
        self.gender = gender

    # Setter & Getter
    def setNIK(self, nik):
        self.nik = nik

    def getNIK(self):
        return self.nik

    def setName(self, name):
        self.name = name

    def getName(self):
        return self.name

    def setGender(self, gender):
        self.gender = gender

    def getGender(self):
        return self.gender
